aea/manager/project.py

```diff
@@ -194,7 +191,7 @@ def _create_private_key(
             if is_connection
             else f"{self.agent_name}_{ledger}_private.key"
         )
```

Comment:
these file names no longer need to be custom now

Reply:
yes